.github/workflows/assign_reviewers_new_contributors.yml

```diff
@@ -0,0 +1,80 @@
+name: Assign Reviewers on PR Creation for New Contributors
+
+on:
+  pull_request:
+    types: [opened]  
+
+jobs:
+  assign-reviewers:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Assign Reviewers and Check for Merged PRs
+        uses: actions/github-script@v4
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const { pull_request, repository } = context.payload;
+            const owner = repository.owner.login;
+            const repo = repository.name;
+            const prNumber = pull_request.number;
+            const username = pull_request.user.login;
+
+            // Fetch all closed PRs of the user
+            const userPRs = await github.pulls.list({
+              owner,
+              repo,
+              state: "closed",
+              per_page: 100,  
+              sort: "created",
+              direction: "desc",
+            });
+
+            const mergedPRs = userPRs.data.filter(pr => pr.merged_at);
+
+            // Check if the user has 0 merged PRs
+            const hasZeroMergedPR = mergedPRs.length === 0;
+
+            if (hasZeroMergedPR) {
```

Comment:
Do the reviewer assignment (and pinging the team if no reviewer from there) for all PRs, not just for new contributors. (We still need to assign reviewers for all PRs.)

For the comment about how to use the PTAL syntax, you just need to show it for new contributors.

Reply:
done